src/cast_core/src/CrudActor.cpp

```diff
@@ -1230,7 +1424,8 @@ CrudActor::CrudActor(genny::ActorContext& context)
     : Actor(context),
       _client{std::move(
           context.client(context.get("ClientName").maybe<std::string>().value_or("Default")))},
-      _loop{context, _client, CrudActor::id()} {}
+      _loop{context, _client, CrudActor::id()},
+      _rng{context.workload().getRNGForThread(CrudActor::id())} {}
```

Comment:
I'm kind of surprised that we need to have a _rng in crudactor now, what do the other implementations currently do (maybe the _rng should be in the stateConfig too).